bandit/formatters/text.py

```diff
@@ -141,29 +132,31 @@ def report(manager, fileobj, sev_level, conf_level, lines=-1):
     """
 
     bits = []
-    bits.append("Run started:%s" % datetime.datetime.utcnow())
 
-    if manager.verbose:
-        bits.append(get_verbose_details(manager))
+    if not manager.quiet or manager.results_count(sev_level, conf_level):
+        bits.append("Run started:%s" % datetime.datetime.utcnow())
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

Naive datetime objects are treated by many datetime methods as local times, We recommend that you create aware timestamps with the `now` or `fromtimestamp` functions using the `tzinfo` to attribute to represent times in UTC. Do not use the [datetime.utcnow()](https://docs.python.org/3/library/datetime.html#datetime.datetime.utcnow) because it returns only a `naive` datetime object. 

[Learn more](https://docs.python.org/3/library/datetime.html#aware-and-naive-objects)